Assignment3/toasterdoom-z170050/query-5-toasterdoom.sparql

```diff
@@ -0,0 +1,8 @@
+PREFIX ont: <http://dbpedia.org/ontology/>
+SELECT DISTINCT ?y COUNT(?z)
```

Comment:
It should be also DISTINCT ?z

Reply:
Do you mean:
SELECT DISTINCT ?y COUNT(DISTINCT ?z)